### ISSUE TITLE
some problem when i use ex_render

### ISSUE THREAD

**Issue by dingxu2016** [joaander/hoomd-examples#1]
when i run your sample : active matter, there is a error : 
<img width="736" alt="issuse" src="https://cloud.githubusercontent.com/assets/19750864/23847941/c06664b0-080f-11e7-8eec-09685596c58d.PNG">

**Comment by joaander** [comment]
ex_render is not a package, it is a helper script I wrote as part of hoomd-examples to display system configurations and trajectories succinctly. It is the file ``ex_render.py`` in the hoomd-examples repository: https://bitbucket.org/glotzer/hoomd-examples/src/852879b73f9256416c148f23ce55f64aba8d5584/ex_render.py?at=master&fileviewer=file-view-default

ex_render uses frensel (https://bitbucket.org/glotzer/fresnel) which depends on tbb. The root cause of your error is that my tbb conda build depended on GLIBC 2.14 and your linux distro does not include that version. 

I have rebuilt my tbb conda build on an older linux distro to remove this requirement. Upgrade with ``conda upgrade -c glotzer tbb`` and the error should no longer occur.
